src/argilla/client/feedback/dataset/base.py

```diff
@@ -371,14 +310,16 @@ def unify_responses(
 
         strategy.unify_responses(self.records, question)
 
+    # TODO(alvarobartt,davidberenstein1957): we should consider having something like
+    # `export(..., training=True)` to export the dataset records in any format, replacing
+    # both `format_as` and `prepare_for_training`
     def prepare_for_training(
```

Comment:
Does make sense to have this functionality linked to local datasets? If the normal workflow is  1. download what you want from argilla locally and 2. train your model, maybe makes sense to prepare the training data only from a local version of the feedback dataset.

Reply:
I've ensured that it's backwards compatible with the current approach, so that the records are automatically pulled from Argilla if we want to use a `FeedbackDataset` in Argilla i.e. `RemoteFeedbackDataset`

Reply:
👍  In any case, I would say to warn users if the `prepare_for_training` method is done using the `RemoteArgillaDataset`. Since this will change in the future, and all the training stuff will be applied using local copies of datasets.